internal/model/table.go

```diff
@@ -231,6 +231,7 @@ func (t *Table) reconcile(ctx context.Context) error {
 	}
 	if err != nil {
 		log.Error().Err(err).Msg("Reconcile failed to list resource")
+		return err
 	}
 
 	var rows render.Rows
```

Comment:
after line #237 Add a check for rows len ie something like this

```go
	if len(oo) > 0 {
		if _, ok := meta.Renderer.(*render.Generic); ok {
			table, ok := oo[0].(*metav1beta1.Table)
			if !ok {
				return fmt.Errorf("expecting a meta table but got %T", oo[0])
			}
			rows = make(render.Rows, len(table.Rows))
			if err := genericHydrate(t.namespace, table, rows, meta.Renderer); err != nil {
				return err
			}
		} else {
			rows = make(render.Rows, len(oo))
			if err := hydrate(t.namespace, oo, rows, meta.Renderer); err != nil {
				return err
			}
		}
	}
```

Reply:
done